odonto/pathways.py

```diff
@@ -5,11 +5,22 @@
 from django.db import transaction
 from opal.core import menus, pathway
 from odonto import models
+from odonto.odonto_submissions import models as submission_models
 from odonto.episode_categories import FP17Episode, FP17OEpisode
 from odonto.odonto_submissions import serializers
 from plugins.add_patient_step import FindPatientStep
 
 
+def get_submission_state(episode):
```

Comment:
This function doesn't get the submission state though... it's `is_submitted` ?

Reply:
👍